fairing/ml_tasks/tasks.py

```diff
@@ -64,8 +65,18 @@ def create(self):
         self.url = self._deployer.deploy(self.pod_spec)
         logger.warning("Prediction endpoint: {}".format(self.url))
 
-    def predict(self, data):
-        r = requests.post(self.url, data=data)
+    def predict_nparray(self, data, feature_names=None):
+        pdata={
+            "data": {
+                "names":feature_names,
+                "tensor": {
+                    "shape": np.asarray(data.shape).tolist(),
+                    "values": data.tolist(),
```

Comment:
data.flatten().tolist()? SeldonMessage proto needs the values to be 1D array https://github.com/SeldonIO/seldon-core/blob/36e96c162bfaa5c1fb5a3286d8c02dba8e393071/util/loadtester/scripts/proto/prediction.proto#L33

Reply:
done